smart_open/asb.py

```diff
@@ -155,14 +155,16 @@ def __init__(
     ):
         if client is None:
             client = azure.storage.blob.BlobServiceClient()
-        self._container_client = client.get_container_client(container)  # type: azure.storage.blob.ContainerClient
+        self._container_client = client.get_container_client(container)
+        # type: azure.storage.blob.ContainerClient
 
         self._blob = self._container_client.get_blob_client(blob)
         if self._blob is None:
-            raise azure.core.exceptions.ResourceNotFoundError('blob {} not found in {}'.format(blob, container))
+            raise azure.core.exceptions.ResourceNotFoundError('blob {} not found in {}'
```

Comment:
Hanging indent please – we don't use vertical indent in `smart_open`. Here and everywhere else.

```suggestion
            raise azure.core.exceptions.ResourceNotFoundError('blob %s not found in %s' % (blob, container))
```